aiida/transports/transport.py

```diff
@@ -185,24 +185,16 @@ def auth_options(cls):  # pylint: disable=no-self-argument
         # The common auth options are currently defined as class members, but the default for `safe_interval` is sub
         # class specific. With the current design the default cannot already be specified directly but has to be added
         # manually here.
-        for option in cls._common_auth_options:
-            if option[0] == 'safe_interval':
-                option[1]['default'] = cls._DEFAULT_SAFE_OPEN_INTERVAL
         return OrderedDict(cls._valid_auth_options + cls._common_auth_options)
 
     @classmethod
     def _get_safe_interval_suggestion_string(cls, computer):  # pylint: disable=unused-argument
         """
-        Default time in seconds between consecutive checks.
+        Return the currently set safe interval.
```

Comment:
I find this confusing because it suggests this is set for a transport instance, but instead it is just for the class. Maybe something like: "Return the default safe interval for this transport class." The second phrase then is not necessary and also a bit misleading. The value is also used in constructing a transport instance if `safe_interval` is not passed in explicitly

Reply:
I agree on the first part, it was a leftover of a first rework of the code I was doing - I realised later that I could have simplified it more. 
Instead, I don't agree on the second - this is *only* used as a suggestion string, while the default in the init is set directly from the `self._DEFAULT_SAFE_OPEN_INTERVAL`:
https://github.com/aiidateam/aiida-core/blob/f68b304b0c7b04245582f8a6f61413dc73929d29/aiida/transports/transport.py#L64